catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/metacard/RegistryUtility.java

```diff
@@ -0,0 +1,127 @@
+     *
+     * @param metacard - will have tags and RegistryId evaluated
+     * @return true if registryId is present in the metacard with the Registry_Tag, false otherwise
+     */
+    public static boolean isRegistryMetacard(Metacard metacard) {
+        if (!metacard.getTags()
+                .contains(RegistryConstants.REGISTRY_TAG)) {
+            return false;
+        }
+
+        Attribute registryAttr = metacard.getAttribute(RegistryObjectMetacardType.REGISTRY_ID);
+        if (registryAttr == null) {
+            return false;
+        }
+
+        String registryId = registryAttr.getValue()
+                .toString();
+
+        return !StringUtils.isEmpty(registryId);
+    }
+
+    /**
+     * Returns the registry id of the metacard passed in. In most cases this method expects that
+     * the user has already evaluated the metacard with the isRegistryMetacard method from this
+     * class.
+     *
+     * @param metacard - the metacard for which the registry Id will be returned
+     * @return the String representation of the metacard's registry id attribute, null if not
+     * present
+     */
+    public static String getRegistryId(Metacard metacard) {
+        return getStringAttribute(metacard, RegistryObjectMetacardType.REGISTRY_ID, null);
+    }
+
+    /**
+     * Identifies whether the metacard is a registry identity node
+     *
+     * @param metacard - metacard to be evaluated for registry identity status
+     * @return true if the metacard is an identity node, false otherwise
+     */
+    public static boolean isIdentityNode(Metacard metacard) {
+        Attribute attribute =
+                metacard.getAttribute(RegistryObjectMetacardType.REGISTRY_IDENTITY_NODE);
+        return attribute != null && (boolean) attribute.getValue();
+    }
+
+    /**
+     * Identifies whether the metacard is a local registry node
+     *
+     * @param metacard - metacard to be evaluated for local registry status
+     * @return true if the metacard is an local node, false otherwise
+     */
+    public static boolean isLocalNode(Metacard metacard) {
+        Attribute attribute = metacard.getAttribute(RegistryObjectMetacardType.REGISTRY_LOCAL_NODE);
+        return attribute != null && (boolean) attribute.getValue();
+    }
+
+    /**
+     * Attempts to return a String attribute's value. If the attribute is null the default value
+     * will be returned instead.
+     *
+     * @param metacard         - metacard to be evaluated for a String type attribute
+     * @param attributeToCheck - attribute to be checked for, should have a String type
+     * @param defaultValue     - the value expected to be returned if the attribute is not present
+     * @return the defaultValue if attribute is not present, otherwise a String representing the
+     * value of the attribute
+     */
+    public static String getStringAttribute(Metacard metacard, String attributeToCheck,
```

Comment:
This method seems more Metacard generic is there a better place for it more associated with Metacard in general?